CHANGES.txt

```diff
@@ -5,6 +5,21 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Updated crate-admin to ``1.1.1`` which includes the following changes:
```

Comment:
I think this could be merged with the previous crate-admin update entry in the unreleased section.